backend/apis/events/get-event.api.js

```diff
@@ -70,12 +68,12 @@ function getEventById(eventId, cbk, connection) {
       return cbk(err, data, fields);
     }
 
-    const bigEventObj = data[0];
```

Comment:
The `data` array doesn't always have at least one object in it (when you specify an invalid event id). This was throwing an error in that case.